src/components/SearchPage.vue

```diff
@@ -39,10 +24,13 @@
         </div>
       </div>
 
-      <!-- forms -->
+      <!-- buttons -->
       <div class="grid-x grid-margin-x button-container">
-        <div class="cell large-8 text-right">
-          <input type="button" value="Done adding symptoms" class="button rounded" :disabled="selectionIsEmpty" @click="goToResults">
+        <div class="cell large-4 text-left">
```

Comment:
Seems like we could also add a template for the buttons as well as a future enhancement.

Reply:
True, that would make a good improvement. If we end up rejiggering the search flow, I'll probably tackle it then.